src/devices/Ina226/Ina226Enum.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Ina226
+{
+    /// <summary>
+    /// An enumeration representting the registers exposed by the INA226 device via I2c.
+    /// </summary>
+    internal enum Ina226Register : byte
+    {
+        /// <summary>Configuration Register r/w</summary>
+        Configuration = 0x00,
+
+        /// <summary>Shunt Voltage Register r</summary>
+        ShuntVoltage = 0x01,
+
+        /// <summary>Bus Boltage Register r</summary>
+        BusVoltage = 0x02,
+
+        /// <summary>Power Register r</summary>
+        Power = 0x03,
+
+        /// <summary>Current Register r</summary>
+        Current = 0x04,
+
+        /// <summary>Calibration Register r/w</summary>
+        Calibration = 0x05,
+
```

Comment:
Seems at least this part is the same as https://github.com/dotnet/iot/blob/main/src/devices/Ina219/Ina219Enum.cs#L113-L128

Reply:
There are a number of differences, I used the ina219 for inspiration but when I tried to tweak it to work with the 226 there were just too many differences.  If it was simplified to the most basic functions then I could probably make something work but I did not want to create a binding that did not expose most all of the features.  the devices functions and specs are very similar but there are a few key differences in their functionality such as the alert modes and pin.

Reply:
I am sure it is possible to share some code between the two bindings but that is still a bit outside of my skill level and the benefit is still somewhat small for only these two.  If someone were to add more of the ina2** family then I could definitely see benefit to combining them.  Something I may try to do in the future once I get my hands on more of the chips to test them on, and more experience with inheritance and interfaces.

Reply:
@krwq Would it be fine to rename the namespace (to e.g Ina2xx) so that the bindings could be merged at least as far as useful (e.g common enums, maybe abstract base)?

Reply:
yes we should rename